packages/back-office-service/prisma/schema.prisma

```diff
@@ -0,0 +1,66 @@
+// This is your Prisma schema file,
+// learn more about it in the docs: https://pris.ly/d/prisma-schema
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+datasource db {
+  provider = "sqlserver"
+  url      = env("DATABASE_URL")
+}
+
+model Project {
+  id                                          Int      @id @default(autoincrement())
+  caseId                                      Int      @unique
+  caseReference                               String?
+  projectName                                 String?
+  projectDescription                          String?
+  publishStatus                               String?
+  sector                                      String?
+  projectType                                 String?
+  sourceSystem                                String
+  stage                                       String?
+  projectLocation                             String?
+  projectEmailAddress                         String?
+  regions                                     String?
```

Comment:
this field (`regions`) is actually an array of strings from Back Office.

Primsa has a JSON type, so we can store it as a stringified array,
or do our own normalisation an build up a table of regions and use
joins.

i am leaning toward json. Any thoughts @Philip-Philippou1 @DavidBeale

Reply:
I think the expectation would be to use a table. Else we would have been allowed to use a Document DB

Reply:
i think the ask was to use sql server rather than nosql rather than avoid any features in sql server that are nosql inspired :) 

i think if we have a known list of regions, we could make this a table. i will check with back office